docs/modelserving/v1beta1/transformer/feast/README.md

```diff
@@ -1,11 +1,161 @@
 # Deploy InferenceService with Transformer using Feast online feature store
 Transformer is an `InferenceService` component which does pre/post processing alongside with model inference. In this example, instead of typical input transformation of raw data to tensors, we demonstrate a use case of online feature augmentation as part of preprocessing. We use a [Feast](https://github.com/feast-dev/feast) `Transformer` to gather online features, run inference with a `SKLearn` predictor, and leave post processing as pass-through.
 
-## Setup
+## Before you begin 
 
 1. Your ~/.kube/config should point to a cluster with [KServe installed](/docs/get_started/README.md#install-the-kserve-quickstart-environment).
 2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
-3. Your Feast online store is populated with driver [data](https://github.com/tedhtchang/driver_rank_repo/blob/master/data/driver_stats.parquet), instructions available [here](https://github.com/tedhtchang/populate_feast_online_store), and network accessible.
+3. You can find the [code samples](https://github.com/kserve/kserve/tree/master/docs/samples/v1beta1/transformer/feast) on kserve repository.
+
+!!! Note
+    This example uses Feast version `0.30.2`
+
+## Create the Redis server
+This example uses the [Redis](https://redis.io/) as the online store. 
+Deploy the Redis server using the below command.
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: redis-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: redis-server
+  template:
+    metadata:
+      labels:
+        app: redis-server
+        name: redis-server
+    spec:
+      containers:
+        - name: redis-server
+          image: redis
+          args: [ "--appendonly", "yes" ]
+          ports:
+            - name: redis-server
+              containerPort: 6379
+          env:
+            - name: ALLOW_EMPTY_PASSWORD
+              value: "yes"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis-service
+spec:
+  type: LoadBalancer
+  selector:
+    app: redis-server
+  ports:
+    - protocol: TCP
+      port: 6379
+      targetPort: 6379
+EOF
+```
+
+!!! Success "Expected output"
+    ```{ .bash .no-copy }
+    $ deployment.apps/redis-server created
+    $ service/redis-service created
+    ```
+
+## Create the Feast server
+
+### Build Feature Store Initializer docker image
+The feature store initializer is a init container which initializes a new sample feature repository, populate the 
+online store with sample driver data and copies the feature repository to the volume mount.
```

Comment:
I guess with init container the features are static and the feast server does not refresh when online store is updated, worth noting that but it is fine as an example